pkg/vm/engine/disttae/logtail_consumer.go

```diff
@@ -1351,12 +1351,19 @@ func updatePartitionOfPush(
 	defer partition.Unlock()
 	v2.LogtailUpdatePartitonGetLockDurationHistogram.Observe(time.Since(t0).Seconds())
 
-	state, doneMutate := partition.MutateState()
+	catalogCache := e.getLatestCatalogCache()
 
-	t0 = time.Now()
-	catache := e.getLatestCatalogCache()
-	key := catache.GetTableById(dbId, tblId)
-	v2.LogtailUpdatePartitonGetCatalogDurationHistogram.Observe(time.Since(t0).Seconds())
+	if !partition.TableInfoOK {
```

Comment:
If there is a DDL alter table, will the tableInfo change?  Do we need to have invalidation code somewhere?

Reply:
It will not change. A new partition will be created.